decidim-core/lib/tasks/decidim_robots.rake

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+namespace :decidim do
+  namespace :robots do
+    desc "Overrides robots.txt with a custom one."
+    task :replace, [] => :environment do
+      actions :append_file, "public/robots.txt", <<~SQUISH
```

Comment:
Why "SQUISH"?? How about something a bit more descriptive, like "EOTASK"?

Reply:
Changed to EOTASK. No particular reason for SQUISH. in my head was similar with https://apidock.com/rails/String/squish